src/validation.cpp

```diff
@@ -5342,6 +5349,14 @@ bool ChainstateManager::PopulateAndValidateSnapshot(
 
     const AssumeutxoData& au_data = *maybe_au_data;
 
+    // This work comparison is a duplicate check with the one performed later in
+    // ActivateSnapshot(), but is done so that we avoid doing the long work of staging
+    // a snapshot that isn't actually usable.
+    if (WITH_LOCK(::cs_main, return !CBlockIndexWorkComparator()(ActiveTip(), snapshot_start_block))) {
+        LogPrintf("[snapshot] activation failed - height does not exceed active chainstate\n");
```

Comment:
In 62ac519e718eb7a31dca1102a96ba219fbc7f95d "validation: do not activate snapshot if behind active chain"

nit: work, not height.

Reply:
> nit: work, not height.

This is addressed in #28562